continuous_integration/envs/38-latest.yaml

```
@@ -14,5 +14,7 @@ dependencies:
   - pytest
   - pytest-cov
   - hilbertcurve
+  - pip:
+    - pymorton
   # optional dependencies
   - pyarrow
```

```suggestion
  # optional dependencies
  - pyarrow
  - pip
  - pip:
    - pymorton
```

It is better to keep pip dependencies on the bottom. Also, as I mentioned before, try to always list `pip` if there are pip installed dependencies.